README.md

```diff
@@ -152,8 +168,27 @@ At this point most users can run the following 4 scripts
 ./bin/add-ccd-roles.sh
 ```
 
-to get their IDAM environment ready and then move on to the [Ready for take-off](###Ready-for-take-off) section.
 
+Create IDAM users and roles
```

Comment:
i think it's CCD roles rathe than `IDAM users and roles`

Reply:
corrected